apps/sms/views/new_message/style/recipients.css

```diff
@@ -79,16 +81,16 @@ Do not remove.
   overflow-y: scroll;
 }
 
+/**
+ * BiDi note: all [contenteditable] blocks must carry a `dir="auto"` attribute,
+ * so we cannot use *-inline-{start,end} properties on them, because the
+ * direction of a dir="auto" element (and therefore the position of start/end)
+ * depends on its content. We want our margin/padding to depend on the document
+ * direction, not element's content.
```

Comment:
but I'm fine with keeping this like this for now, and looking at this in a follow-up. I'd rather see this in sooner than later.